pyanaconda/modules/storage/iscsi/iscsi.py

```diff
@@ -220,6 +220,10 @@ def get_dracut_arguments(self, node):
 
         blivet_node = iscsi.get_node(node.name, node.address, node.port, node.iface)
 
+        if not blivet_node:
```

Comment:
Isn't there a unit test that could be extended?

Reply:
I haven't found any (there is only a simple one for fcoe), but I'll add one.